ray-operator/test/migration/README.md

```
@@ -0,0 +1,241 @@
+   - Custom environment variables
+   - Custom volumes and volume mounts
+   - Custom ServiceAccountName (non-CodeFlare patterns)
+
+## Test RayClusters
+
+The `testdata/` directory contains various RayCluster configurations:
+
+### Generic Test Cases
+
+| File | Description |
+|------|-------------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)